specs/altair/sync-protocol.md

```diff
@@ -276,21 +349,25 @@ def validate_light_client_update(store: LightClientStore,
     assert bls.FastAggregateVerify(participant_pubkeys, signing_root, sync_aggregate.sync_committee_signature)
 ```
 
-#### `apply_light_client_update`
+### `apply_light_client_update`
 
 ```python
 def apply_light_client_update(store: LightClientStore, update: LightClientUpdate) -> None:
     store_period = compute_sync_committee_period_at_slot(store.finalized_header.slot)
     update_finalized_period = compute_sync_committee_period_at_slot(update.finalized_header.slot)
-    if update_finalized_period == store_period + 1:
+    if not is_next_sync_committee_known(store):
+        assert update_finalized_period == store_period
```

Comment:
Is there an edge-case, that all the incoming updates are of newer finalized periods? e.g., the initial bootstrap is not fresh enough.

Reply:
No additional edge-case is introduced, this is just a way to obtain the initial sync committee. From there on, historic light client updates need to be applied, same way as before. Or, alternatively, regular forward sync can be used (based on the current finalized state root / block root) to proceed to a newer sync committee period for which data is available.

Reply:
Also, an older `LightClientUpdate` for the current store period is accepted to obtain the `next_sync_committee`, if the question is referring for that case.